managed/ui/src/components/tables/ListBackups/ListBackups.js

```diff
@@ -385,6 +392,13 @@ export default class ListBackups extends Component {
       })
       .filter(Boolean);
 
+    // Check for in-progress backups and mark them as unselctable.
+    selectRowProp.unselectable = backupInfos
+      .filter((backup) => {
+        return backup.status === 'InProgress';
+      })
+      .map((data) => data.backupUUID);
```

Comment:
just for consistency this would be better
```suggestion
      .filter((backup) => backup.status === 'InProgress')
      .map((data) => data.backupUUID);
```

Reply:
Done